kuma/wiki/templates/wiki/new_document.html

```diff
@@ -65,6 +65,16 @@
         </div>
       {% endif %}
 
+      <section id="page-tags" class="page-tags wiki-block">
+        <h3><i aria-hidden="true" class="icon-tags"></i>{{ _('Tags') }} <a href="{{ wiki_url('Project:MDN/Contributing/Editor_guide/Editing#The_tags_box') }}"><i aria-hidden="true" title="{{ _('Learn how to tag an article') }}" class="icon-question-sign editor-help-icon"></i></a></h3>
+        <p>
+        {% trans url=wiki_url('Project:MDN/Contributing/Tagging_standards') %}
+        Categorize the article. It will make the article findable under the right filters in the search engine. <a target="_blank" href="{{ url }}">Read the Tagging Guide</a>.
+        {% endtrans %}
+        </p>
+        <input id="tagit_tags" type="text" name="tags" value="{% for tag in tags %}{{ tag.name }},{% endfor %}" maxlength="255" class="tags" />
+      </section>
+
       {% include 'wiki/includes/revision_comment.html' %}
 
       {% include 'wiki/includes/review_tags.html' %}
```

Comment:
We already have the tags form on this page, the code you added should replace the review_tags include otherwise we prompt for tags twice on the page ;)